packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -101,7 +106,8 @@ const CreateAdForm = () => {
                     max_transaction: '',
                     min_transaction: '',
                     offer_amount: '',
-                    price_rate: '',
+                    payment_info: my_ads_store.payment_info,
+                    price_rate: floating_rate_store.rate_type === ad_type.FLOAT ? '-0.01' : '',
```

Comment:
price_rate isn't clear, please change it to rate_type.

Reply:
Will need to change in a lot of places. Please advice

Reply:
Made the change